src/backend/avatarGenerator.ts

```diff
@@ -0,0 +1,13 @@
+export const generateAvatar = (name: string, variant: string = 'marble', size: number = 120, colors: string[] = []): string => {
```

Comment:
Are you sure that once the URL is generated and saved into database, it will be static and not changed?